apps/librepcb/main.cpp

```diff
@@ -205,7 +205,22 @@ static bool isFileFormatStableOrAcceptUnstable() noexcept {
  ******************************************************************************/
 
 static FilePath determineWorkspacePath() noexcept {
-  FilePath wsPath(Workspace::getMostRecentlyUsedWorkspacePath());
+  static const QString FORCE_VAR_NAME("LIBREPCB_WORKSPACE");
+  FilePath wsPath;
+  QString wsForceStr = qgetenv(FORCE_VAR_NAME.toStdString().c_str());
+
+  if (wsForceStr != "") {
+    wsPath = FilePath(wsForceStr);
+    if (!Workspace::isValidWorkspacePath(wsPath)) {
+      qWarning() << FORCE_VAR_NAME << "set to invalid workspace:" << wsForceStr;
+      wsPath = FilePath("");
```

Comment:
Not really critical, but I'd recommend to use the default constructor to explicitly create an invalid FilePath object:

```suggestion
      wsPath = FilePath();
```